x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_definitions/entity_descriptions/generic.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { newestValue } from './field_utils';
+import type { EntityDescription } from '../types';
+import { getCommonFieldDescriptions } from './common';
+
+export const GENERIC_DEFINITION_VERSION = '1.0.0';
+export const GENERIC_IDENTITY_FIELD = 'entity.id';
```

Comment:
I'm wondering if only the `entity.id` can guarantee uniqueness since entity.id can be a `host.name`, and also a `user.name`, if a user and a host have the same name somehow they would be considered the same entity? This might be an unlikely example, but it seems to me it can be very likely to happen for other entity types (database, bucket, function, etc) doesn't it? Should the identity field be a combination of `entity.id` + `entity.type`?